src/pybind/mgr/dashboard/frontend/src/app/shared/components/cd-label/cd-label.component.ts

```diff
@@ -0,0 +1,47 @@
+import { AfterContentInit, Component, ElementRef, Input, ViewChild } from '@angular/core';
+
+import { FixedColorClass } from '~/app/shared/enum/fixed-color-class.enum';
+
+@Component({
+  selector: 'cd-label',
+  templateUrl: './cd-label.component.html',
+  styleUrls: ['./cd-label.component.scss']
+})
+export class CdLabelComponent implements AfterContentInit {
+  @ViewChild('contentWrapper')
+  content_child: ElementRef;
```

Comment:
I like snake_case over CamelCase, but Angular is all CamelCase, so this should be:
```suggestion
  contentChild: ElementRef;
```